.github/workflows/preview-build.yml

```diff
@@ -45,6 +45,7 @@ permissions:
 
 jobs:
   build:
+    if: github.event.repository.fork == false
```

Comment:
Don't we still want to ensure the docs build but only skip the preview environment?

Reply:
IMO, putting the condition here makes it simple.

If we only skip the preview environment the logic becomes more complex. 
It already is pretty complex with all the logical conjunctions.

And I don't think we have a use case for running the build on a forked repository.

I think we should only do it if there is reason for it.

Reply:
We want to ensure the documentation has no build errors and all the links validate for forked PR's as well.

Reply:
Do you know if there are devs/techwriters creating PRs own their own fork as part of their daily workflow?

IMHO, what we care about is the PR on the upstream repo.

Reply:
Ahh I thought this skipped the job from running on PR builds from forks to the main repository. 

Are you saying this disables the job from running on PR's TO a forked repository only?

Reply:
Yes. 

I did not test it.. I can still do it though.

But this was also the reason for the original draft in https://github.com/elastic/kibana/pull/214886.

@Ikuni17 wanted to disable the run on his own fork.

Reply:
Roger that! maybe add a comment to highlight this?

Reply:
https://github.com/elastic/docs-builder/pull/763/commits/4e4938dc356efbe417053d501b64c99ab376333f